README.md

```diff
@@ -69,8 +69,10 @@
 - [FreeCodeCamp](https://github.com/freeCodeCamp/freeCodeCamp)
 - [Funny Algorithms](https://github.com/ReciHub/FunnyAlgorithms)
 - [Hackerrank-codes](https://github.com/swapnanildutta/Hackerrank-Codes)
+- [HacktoberFest2020-Contributions](https://github.com/bhaveshlohana/
```

Comment:
I think we got some conflict here, can you fix it?

Reply:
May I know what happened?